Lib/fontTools/otlLib/builder.py

```diff
@@ -471,6 +839,15 @@ def add_subtable_break(self, location):
 
 
 class ClassPairPosSubtableBuilder(object):
+    """Builds class-based Pair Positioning (GPOS2 format 2) subtables.
+
+    Note that this does *not* build a GPOS2 ``otTables.Lookup`` directly,
+    but builds a list of ``otTables.PairPos`` subtables. It is used by the
+    :class:`PairPosBuilder` below.
+
+    Attributes:
```

Comment:
should we also document the other two init parameters `valueFormat1` and `valueFormat2` here?

Reply:
I can't see that they're actually used. I'm going to remove them instead...

Reply:
Interesting. I wonder why they were there in the first place..